src/components/icon-search-popover/index.js

```diff
@@ -71,6 +71,10 @@ export const cleanSvgString = svgString => {
 		newSvg = newSvg.replace( /\s*<g\s*>([\s\S]*?)<\/g>\s*/gm, '$1' )
 	}
 
+	// Remove width and height attribute so that we can control the size (default to 100%)
+	newSvg = newSvg.replace( /width="[^"]*"/, '' )
+		.replace( /height="[^"]*"/, '' )
```

Comment:
1. Can you replace attributes only if they are inside the `<svg>` tag, so we can be sure that other tags won't be affected.
2. Please support cases where single quotes are used e.g. `width='100'`
```suggestion
	newSvg = newSvg.replace( /width="[^"]*"/, '' )
		.replace( /height="[^"]*"/, '' )
```